cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -646,12 +684,13 @@ func create(ctx context.Context, flags *pflag.FlagSet) error {
 			}
 		}
 
+		endpointList = []string{fmt.Sprintf("%s:%d", forceEndpoint, apiPort)}
 		genOptions = append(genOptions, generate.WithEndpointList(endpointList))
 		configBundleOpts = append(configBundleOpts,
 			bundle.WithInputOptions(
 				&bundle.InputOptions{
 					ClusterName: clusterName,
-					Endpoint:    fmt.Sprintf("https://%s", nethelpers.JoinHostPort(defaultInternalLB, controlPlanePort)),
+					Endpoint:    fmt.Sprintf("https://%s", nethelpers.JoinHostPort(defaultEndpoint, externalControlPlanePort)),
```

Comment:
Unsure why `defaultInternalLB` was here and what could possibly break

Reply:
this is in-cluster Kubernetes endpoint (not outside of cluster): 

* so in case of docker it is `10.5.0.2:6433` (first controlplane container:6443)
* in case of QEMU it's `10.5.0.1:6443` loadbalancer listening on the bridge

we shouldn't be changing it

Reply:
the way this thing works is that inside the cluster the endpoint is "normal", but `ForceEndpoint` overrides the IP address of the endpoint when we pull `kubeconfig` out of the user.

So in case of Docker, with plain defaults it looked like that (before your changes):

* inside cluster: `https://10.5.0.2:6443`
* outside (for the user): `https://127.0.0.1:6443`

With your changes, "inside" should stay the same, while outside will be `https://127.0.0.1:<dynamic_allocated_port>`